package.json

```diff
@@ -51,6 +51,7 @@
     "inherits": "2.0.1",
     "jade": "pugjs/jade#29784fd",
     "jed": "1.0.2",
+    "jshashes": "1.0.5",
```

Comment:
I think we need to do something with shrinkwrap when we update the base `package.json`. /cc @gwwar